scheduler.py

```diff
@@ -0,0 +1,41 @@
+"""scheduler used by the sndtrace_cp
+
+This module is intended to replace the usage of the napp
+called scheduler soon to be deprecated.
+"""
+import pytz
+from apscheduler.jobstores.base import JobLookupError
+from apscheduler.schedulers.background import BackgroundScheduler
+from kytos.core import log
+
+
+class Scheduler:
+    """Class to set scheduler."""
+
+    def __init__(self) -> None:
+        """Create an instance of Scheduler and starts it."""
+        self.scheduler = BackgroundScheduler(timezone=pytz.utc)
+        self.scheduler.start()
+
+    def add_job(self, settings):
```

Comment:
Got it. About the arguments from `add_callable()`, `trigger` is causing problems with `**trigger_args`. `{'trigger': trigger}` is being taken out and stored in the variable `trigger` then `**trigger_args` ends up with just `{'seconds':20}` instead of `{'trigger': trigger, 'seconds':20}`.

Reply:
It could be, however, it could turn out to be too limiting for the purpose of this scheduler that's reponsible for scheduling callables. As of now the callables that this NApp will schedule `run_traces` and `run_important_traces` don't take args or kwargs so it would work, but to empower this scheduler to be capable of scheduling callables and tiny changes that these methods might have args or kwargs in the future to not require refactoring, that's why I suggested `args` and `kwargs` in addition to `**trigger_args` to be compatible with the [underlying `add_job` signature](https://apscheduler.readthedocs.io/en/stable/modules/schedulers/base.html#apscheduler.schedulers.base.BaseScheduler.add_job) without too much extra effort from this abstraction, so `args=None` and `kwargs=None` accordingly to know the underlying library work.

Reply:
Got it